submissions/kirill8210/popup-task/main.css

```diff
@@ -0,0 +1,236 @@
+html{
+    box-sizing: border-box;
+}
+
+*, *::before, *::after{
+    box-sizing: inherit;
+}
+
+body{
+    font-family: Roboto, sans-serif;
+    line-height: 1.3;
+    font-weight: 400;
+}
+
+img{
+    max-width: 100%;
+    height: auto;
+}
+
+button{
+    cursor: pointer;
+    padding: 0;
+}
+
+a{
+    text-decoration: none;
+    color: inherit;
+}
+
+ul{
+    list-style: none;
+    padding: 0;
+    margin: 0;
+}
+
+h1, h2, h3, p{
+    margin: 0;
+}
+
+.container{
+    min-height: 100vh;
+    max-width: 1200px;
+    margin: 0 auto;
+    padding: 0 30px;
+}
+
+.header{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.menu{
+    display: flex;
+    align-items: center;
+    padding: 10px 0;
+    position: relative;
+}
+
+.menu > li{
+    display: flex;
+    align-items: center;
+    margin-left: 10px;
+}
+
+.menu > li{
+    border: 1px solid transparent;
+    height: 35px;
+}
+
+.menu > li:hover{
+    border: 1px solid #e5e5e5;
+}
+
+.menu > li > a{
+    display: flex;
+    align-items: center;
+    height: 25px;
+    padding: 5px;
+}
+
+.menu > li > label{
+    display: flex;
+    align-items: center;
+    width: 35px;
+    height: 35px;
+    padding: 5px;
+    border: 1px solid transparent;
+}
+
+.checkbox, .checkbox_more{
+    appearance: none;
+    position: absolute;
+}
+
+.checkbox:checked ~ .popup, .checkbox_more:checked ~ .popup_more{
+    display: flex;
+}
+
+.checkbox:checked ~ label{
+    border: 2px solid #a6c8ff;
+}
+
+.checkbox_more:checked ~ label span{
+    border: 2px solid #a6c8ff;
+    width: 294px;
+}
+
+.popup_img{
+    width: 25px;
+}
+
+.popup{
+    display: none;
+    position: absolute;
+    top: 50px;
+    right: 0;
+    width: 310px;
+    height: 453px;
+    overflow: auto;
+    flex-flow: wrap;
+    gap: 0 10px;
+    box-shadow: 0 2px 8px rgba(0, 0, 0, .25);
+}
+
+.popup a, .popup_more label{
+    font-size: 14px;
+    line-height: 16px;
+}
+
+.popup img, .popup_more img{
+    max-width: 45px!important;
```

Comment:
Why do you need important here? Try to avoid it, it is anti-pattern [read more](https://uxengineer.com/css-specificity-avoid-important-css/)

Reply:
ok

Reply:
You don't fixed, please, check all css file and avoid !important

Reply:
ok